webapp/src/components/reaction_stream/reaction_stream.tsx

```diff
@@ -58,6 +67,28 @@ export const ReactionStream = (props: Props) => {
             </ReactionChip>
         );
     });
+
+    // add hands up
+    let elements = [];
+    if (props.handsup?.length) {
+        const handsupElement = (<Emoji emoji={{name: 'hand', skin: '', unified: '270B'}}/>);
+        const first = props.handsup[0] === props.currentUserID ? 'You' : getUserDisplayName(props.profiles[props.handsup[0]]);
+
+        // maybe we can consider having 1, 2 or more participants as a stretch
+        const participants = props.handsup?.length > 1 ? `${first} & ${props.handsup?.length - 1} others` : first;
```

Comment:
Should do three cases here, since if theres 2 reactions we will have "x and 1 others" which is grammatically incorrect